Tesla.css

```diff
@@ -0,0 +1,214 @@
+* {
+    margin: 0;
+    padding: 0;
+  }
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)